.prettierignore

```diff
@@ -10,6 +10,7 @@ coverage/
 
 # Build output
 dist/
+release/
```

Comment:
Interesting, do we have a base directory level `release` folder anywhere at the mo?

Reply:
Nah we don't but this matches `/package/release/` too

Reply:
We're going to [keep `./dist` at project root](https://github.com/alphagov/govuk-frontend/pull/3491#discussion_r1173953748) for now